apps/docs/content/components/CTABanner.mdx

```diff
@@ -0,0 +1,205 @@
+---
+title: CTABanner
+status: Experimental
+figma: 'https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=539%3A14285&t=RJEWZhVUtrRQ0pKh-0'
+source: https://github.com/primer/brand/tree/main/src/CTABanner/CTABanner.tsx
+storybook: '/brand/storybook/?path=/docs/components-ctabanner--playground'
+description: Use CTABanner to create call to action content in a separated section.
+---
+
+```js
+import {CTABanner, ButtonGroup, Button} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<CTABanner>
+  <CTABanner.Heading>
+    Where the most ambitious teams build great things
+  </CTABanner.Heading>
+  <CTABanner.Description>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
+    ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus
+    elementum.
+  </CTABanner.Description>
+  <ButtonGroup buttonSize="large">
+    <Button>Primary Action</Button>
+    <Button>Secondary Action</Button>
+  </ButtonGroup>
+</CTABanner>
+```
+
+### Aligned center
+
+The content alignment can be changed using the `align` prop on the root `<ComparisonTable>`. This can be configured to either `start` or `center`.
+
+```jsx live
+<CTABanner align={'center'}>
+  <CTABanner.Heading>
+    Where the most ambitious teams build great things
+  </CTABanner.Heading>
+  <CTABanner.Description>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
+    ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus
+    elementum.
+  </CTABanner.Description>
+  <ButtonGroup buttonSize="large">
+    <Button>Primary Action</Button>
+    <Button>Secondary Action</Button>
+  </ButtonGroup>
+</CTABanner>
+```
+
+### Optional border
+
+A border can be provided using the `hasBorder` prop on the root `<ComparisonTable>`. This will render a border around the CTABanner component giving separatoin between the foreground and background. Especially when there is no shadow present on the background.
+
+```jsx live
+<CTABanner hasBorder>
+  <CTABanner.Heading>
+    Where the most ambitious teams build great things
+  </CTABanner.Heading>
+  <CTABanner.Description>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
+    ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus
+    elementum.
+  </CTABanner.Description>
+  <ButtonGroup buttonSize="large">
+    <Button>Primary Action</Button>
+    <Button>Secondary Action</Button>
+  </ButtonGroup>
+</CTABanner>
+```
+
+### Optional background
+
+A background can be removed using the `hasBackground` prop on the root `<ComparisonTable>`. This will render the CTABanner component without a background allowing it to blend in with its parent.
+
+```jsx live
+<CTABanner hasBackground={false}>
```

Comment:
![example lacking contrast between foreground and background](https://user-images.githubusercontent.com/13340707/204800503-ca952ade-18d5-4a06-ab43-7e3f56c96e32.png)

Not sure if this is the example I'd expect given the heading. 

Can we either provide a custom image or background color (maybe something from base scale) in the background or change the title to say "remove background fill" or similar. cc. @danielguillan for other ideas.

Also that shadow should probably be turned off here, it's a bit much 😀

Reply:
Oh yes so after the chat we had last week the docs were supposed to only show this with the shadow and background removed. So I will also remove the shadow. We discussed linking the two potentially if that proves to be how it is used most.